src/filter/levels/levels.c

```diff
@@ -125,9 +148,9 @@ f0r_instance_t f0r_construct(unsigned int width, unsigned int height)
   inst->outputMin = 0;
   inst->outputMax = 1;
   inst->gamma = 1;
-  inst->channel = 0;
+  inst->channel = CHANNEL_RED;
```

Comment:
Ideally, I'd like to change this to `CHANNEL_LUMA` as this is is imho a more useful default. However, this would change the default values from before (though it _should_ not make any difference), so I left it at `CHANNEL_RED`  for now.

Reply:
I agree. You can change it and increment the minor version in the plugin_info.

Reply:
Done. Changed to LUMA, increased minor version +=1